packages/web/src/pages/search-page/search-results/ProfileResults.tsx

```diff
@@ -26,14 +25,36 @@ const messages = {
 }
 
 type ProfileResultsProps = {
-  queryData: Omit<FlatUseInfiniteQueryResult<UserMetadata>, 'status'>
+  // the 'status' type was conflicting with the data we pass from useSearchAllResults - but we don't use it at all here so no need to worry about it
+  queryData: Omit<ReturnType<typeof useSearchUserResults>, 'status'>
   limit?: number
   skeletonCount?: number
 }
 
-export const ProfileResults = (props: ProfileResultsProps) => {
-  const { limit = 100, skeletonCount = 10, queryData } = props
-  const { data = [] } = queryData
+const ProfileResultsSkeletons = ({
+  skeletonCount = 12
+}: {
+  skeletonCount: number
+}) => {
+  const isMobile = useIsMobile()
+  return (
+    <>
+      {range(skeletonCount).map((_, i) => (
+        <UserCard
+          key={`user_card_sekeleton_${i}`}
+          id={0}
+          size={isMobile ? 'xs' : 's'}
+          css={isMobile ? { maxWidth: 320 } : undefined}
+          loading={true}
+        />
```

Comment:
feels like maybe we should extract the skeleton from UserCard and import it directly here?